crates/nomination/src/lib.rs

```diff
@@ -162,6 +162,7 @@ pub mod pallet {
         #[pallet::weight(<T as Config>::WeightInfo::opt_out_of_nomination())]
         #[transactional]
         pub fn opt_out_of_nomination(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
+            ext::security::ensure_parachain_status_running::<T>()?;
```

Comment:
Why don't we allow this call when the oracle is offline?

Reply:
We will introduce force refunds in a subsequent PR that will be dependent on the price oracle to ensure vaultd remain over-collateralized.